wp-content/themes/rebalance-child-mmn/functions.php

```diff
@@ -246,6 +246,80 @@ function suppress_feature_image(){
   }
 }
 
+/*mmn: merites blog conditional formatting*/
+function add_custom_font() {
+	//mmn: add custom font for merites
+    $font_url = get_stylesheet_directory_uri() . '/merites/WindsorConDExtBol.otf';
+    $font_family = 'WindsorConDExtBol';
+
+    $font_formats = array(
+        'otf' => 'opentype',
+    );
+
+    $font_style = '@font-face {
+        font-family: "' . $font_family . '";
+        src: url("' . esc_url($font_url) . '") format("' . implode(', ', $font_formats) . '");
+        font-weight: normal;
+        font-style: normal;
+    }';
+
+    wp_add_inline_style('rebalance-style', $font_style);
+}
+add_action('wp_enqueue_scripts', 'add_custom_font');
+
+function is_merites() {
+	//mmn: helper function for deciding on rendering the normal site or merites subblog
+	//returns true if standing on an article written by the user 'merites'
+	//or if standing on the author page (/author/merites)
+	$merites_path = get_stylesheet_directory_uri() . '/merites';
+	$author_username = get_the_author_meta('user_nicename', get_post_field('post_author', get_the_ID()));
+	if (is_author('merites') || ( is_single() && $author_username == 'merites' )) {
```

Comment:
The user_nicename field works for me in this latest state. @gammaw can you check?

Reply:
Looks good! Thanks!